src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -303,14 +333,61 @@ private void exportMembers(final File file) {
             logger.warn("Unable to parse file: {}", ex.toString());
         }
     }
-    void writeResponse(final FcrepoResponse response, final List<URI> describedby, final File file)
+
+    /**
+     * Filter out the binary resource references from the model
+     * @param model
+     * @return
+     * @throws FcrepoOperationFailedException
+     * @throws IOException
+     */
+    private Model filterBinaryReferences(final URI uri, final Model model) throws IOException,
+            FcrepoOperationFailedException {
+
+        final List<Statement> removeList = new ArrayList<>();
+        for (final StmtIterator it = model.listStatements(); it.hasNext();) {
+            final Statement s = it.nextStatement();
+
+            final RDFNode obj = s.getObject();
+            if (obj.isResource() && obj.toString().startsWith(repositoryRoot.toString())
+                    && !s.getPredicate().toString().equals(REPOSITORY_NAMESPACE + "hasTransactionProvider")) {
+                try (final FcrepoResponse resp = client().head(URI.create(obj.toString())).disableRedirects()
+                        .perform()) {
+                    checkValidResponse(resp, uri, config.getUsername());
+                    final List<URI> linkHeaders = resp.getLinkHeaders("type");
+                    if (linkHeaders.contains(binaryURI)) {
+                        removeList.add(s);
+                    }
+                }
+            }
+        }
+
+        model.remove(removeList);
+        return model;
+    }
+
+    /**
+     * Method to find and set the repository root from the resource uri.
+     * @param uri the URI for the resource
+     * @throws IOException
+     * @throws FcrepoOperationFailedException
+     */
+    private void findRepositoryRoot(final URI uri) throws IOException, FcrepoOperationFailedException {
```

Comment:
Can we consolidate this method and:
https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/main/java/org/fcrepo/importexport/importer/Importer.java#L651